forestserviceprototype/specialuseform/templates/specialuseform/print_permit.html

```diff
@@ -0,0 +1,57 @@
+{% load static from staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <title>{% block title %}Forest Service Prototype{% endblock %}</title>
+      <link rel="stylesheet" href="{% static "additional.css" %}">
+  </head>
+  <body>
```

Comment:
Could we add some kind of instructions at the top, like "**Print this page and take it with you during your activity**?"